content/en/blog/_posts/2019-06-05-what-and-the-why-of-cluster-api.md

```diff
@@ -0,0 +1,63 @@
+
+### Goals
+
+
+* The cluster management APIs should be declarative, Kubernetes-style APIs that follow our existing API Conventions.
+* To the extent possible, we should separate state that is environment-specific from environment-agnostic. However, we still want the design to be able to utilize environment-specific functionality, or else it likely won’t gain traction in favor of other tooling that is more powerful.
+
+### Non-goals
+
+* To add these cluster management APIs to Kubernetes core.
+* To support infrastructure that is irrelevant to Kubernetes clusters. We are not aiming to create Terraform-like capabilities of creating any arbitrary cloud resources, nor are we interested in supporting infrastructure used solely by applications deployed on Kubernetes. The goal is to support the infrastructure necessary for the cluster itself.
+* To convince every Kubernetes lifecycle product (kops, Kubespray, Amazon Elastic Container Service for Kubernetes, Azure Kubernetes Service, Google Kubernetes Engine, VMware Cloud PKS, VMware Enterprise PKS, VMware Essential PKS, etc.) to support these APIs. There is value in having consistency between installers and broad support for the cluster management APIs and in having common infrastructure reconcilers used post-installation, but 100% adoption isn’t an immediate goal.
+* To model state that is purely internal to a deployer. Many Kubernetes deployment tools have intermediate representations of resources and other internal state to keep track of. They should continue to use their existing methods to track internal state, rather than attempting to model it in these APIs.
+
+## State of Cluster API
+
+We are planning an initial v1alpha1 release of Cluster API near the end of March 2019. This will be our first release, and fully expect there to be several iterations before we find a solution that the community is happy with. That said, the community is vibrant and growing, and for those who are interested in cluster management, there are [ample opportunities to engage with the community](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle). Come join our [mailing lists](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle) and [Slack channels](https://kubernetes.slack.com/messages/sig-cluster-lifecycle) for more information.
```

Comment:
nit: March 2019 has now passed.

Reply:
indeed, i just kept it as per the original content. the post comes a bit late on K8s side but then again whoever will read i think will know/ understand when the initial release was cut, you agree ?